eng/packages/http-client-csharp/generator/Azure.Generator/test/Providers/CollectionResultDefinitions/ContinuationTokenTests.cs

```diff
@@ -0,0 +1,148 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Generator.Providers;
+using Azure.Generator.Tests.Common;
+using Azure.Generator.Tests.TestHelpers;
+using Microsoft.TypeSpec.Generator.Input;
+using Microsoft.TypeSpec.Generator.Primitives;
+using NUnit.Framework;
+using System.Linq;
+
+namespace Azure.Generator.Tests.Providers.CollectionResultDefinitions
+{
+    public class ContinuationTokenTests
```

Comment:
Can we also adopt the spector scenarios https://github.com/Azure/azure-sdk-for-net/issues/49713

Reply:
I will leave this for now, need to continue with MPG generator for higher priority.